app/aboutUs/index.js

```diff
@@ -0,0 +1,17 @@
+var angular = require('angular'),
```

Comment:
The angular variable is added as a global variable in .eslintrc.yml, we don't have to write this any more. You can remove the first line.